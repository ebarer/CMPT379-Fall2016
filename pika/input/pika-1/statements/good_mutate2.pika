exec {
	var c := 3.2.
	const d := 8 * 5.
	print c; d, _n_.
	var e := 1.333.
	c := 4.9.
	print c; d; e, _n_.
	var f := 2.818.
	c := 0.0.
	print c; d; e; f, _n_.
	var g := 3.2E3.
	c := 77.77.
	print c; d; e; f; g, _n_.
}
