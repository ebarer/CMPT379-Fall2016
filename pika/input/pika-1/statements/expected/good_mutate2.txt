3.2 40
4.9 40 1.333
0 40 1.333 2.818
77.77 40 1.333 2.818 3200
